### DISCUSSION TITLE
Welcome to df-aggregator Discussions!

### DISCUSSION THREAD

**Discussion by ckoval7** [ckoval7/df-aggregator#1] [General]
Feel free to post feedback and general questions here. If you're encountering an error or notice a bug, please open an issue.

**Comment by ghost** [comment]
Hey Corey,
Thanks for sharing the df-aggregator and the fork of kerberossdr with the gpsd support. I have the gpsd fork installed and will test it tomorrow on the gps receiver that is attached to my APRS system in my vehicle.

My problem is viewing the web page on df-aggregator. I have it installed and can run it but I'm not sure how to configure it so I can see the additional web pages.
Can you shed some light?

Thanks,
John

**Comment by aaronsheldon** [comment]
I was wondering if there would be any interest in a Bayesian updating algorithm to sequentially generate the "best point" based on the real-time sequence of measurements of position, time, and altitude of the SDR and direction of the multi-pole maximum lobe? I have been mulling this over for a drone based platform to combine with a Bubble.io hosted REST API. The math I'm chewing through is a 2-D Guassian posterior on sphere representing the distribution of the estimate of the broadcast position combined with an initial uniform prior (potentially over the surface of the earth). Each iteration would take the previous iterations estimate of the mean and covariance matrix of the posterior and update it with the current observed direction from a given position and time. This could be used to draw a "3 sigma" confidence ellipsoid around the estimate of the mean broadcast position. I can definitely get this to work for static broadcasters, and might be able to work out the estimator for broadcasters moving "linearly" on great arcs, or possibly piece-wise quadratic (position, direction of motion, direction of acceleration). But that will take a bit of tensor magic. Thoughts?

**Reply by alphafox02** [reply]
Sounds interesting, this is just applied server side to calculate what’s being sent by KerberosSDr or KrakenSDR? I once had some friends fly a KerberosSDr on a drone as a test, but never finished where I was going with it.
